### ISSUE TITLE
Cannot use 2 font families

### ISSUE THREAD

**Issue by sebastiangrebe** [vercel/next.js#50625]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
Platform: macOS
Version: Big Sur
Binaries:
Node: 16.16.0
npm: 8.11.0
Yarn: 1.22.17
Relevant packages:
next: 13.1.1
eslint-config-next: 13.4.4
react: 18.2.0
react-dom: 18.2.0
```

### Which area(s) of Next.js are affected? (leave empty if unsure)

Font optimization (next/font)

### Link to the code that reproduces this issue or a replay of the bug

https://stackblitz.com/edit/nextjs-fonts

### To Reproduce

We are using below config where the first one is for latin characters in english and the second one is for arabic characters.

```
import localFont from 'next/font/local';

export const myProxima = localFont({
  variable: '--my-proxima',
  display: 'swap',
  adjustFontFallback: false,
  src: [
    {
      path: './Proxima-Nova-Bold.woff2',
      weight: '700',
      style: 'normal',
    },
    {
      path: './Proxima-Nova-Light.woff2',
      weight: '300',
      style: 'normal',
    },
    {
      path: './Proxima-Nova-Regular-Italic.woff2',
      weight: '400',
      style: 'italic',
    },
    {
      path: './Proxima-Nova-Regular.woff2',
      weight: '400',
      style: 'normal',
    },
    {
      path: './Proxima-Nova-Semibold.woff2',
      weight: '600',
      style: 'normal',
    },
    {
      path: './Proxima-Nova-Thin-Italic.woff2',
      weight: '200',
      style: 'italic',
    },
    {
      path: './Proxima-Nova-Thin.woff2',
      weight: '200',
      style: 'normal',
    },
  ],
});

export const myGESS = localFont({
  variable: '--my-gess',
  display: 'swap',
  adjustFontFallback: false,
  src: [
    {
      path: './GESSTwoLight.woff',
      weight: '200',
      style: 'normal',
    },
    {
      path: './GESSTwoLight.woff',
      weight: '300',
      style: 'normal',
    },
    {
      path: './GESSTwoLight.woff',
      weight: '400',
      style: 'normal',
    },
    {
      path: './GESSTwoMedium.woff',
      weight: '600',
      style: 'normal',
    },
    {
      path: './GESSTwoBold.woff',
      weight: '700',
      style: 'normal',
    },
  ],
});

```

### Describe the Bug

Normally we can just rely on font-face to automatically switch the font to the one which is able to handle all characters in a displayed string.
After starting to use next/font we need to pass the prop `adjustFontFallback: true` as otherwise the english font would always over take the arabic font as the fallback will be always applied.
Similarly if we don't add it on arabic we would start displaying arabic numbers instead of latin numbers which in our use case is also unconventional.

For now we are handling it via adjustFontFallback but that defeats the purpose of improving our CLS.

Same issue was reported here but closed: https://github.com/vercel/next.js/issues/44825

### Expected Behavior

Able to add a fallback which acts the same way as the font that was assigned so that we can handle such cases via normal font-face behavior of the browser to use the font-family which is able to handle all characters of string.

### Which browser are you using? (if relevant)

Chrome 113.0.5672.126

### How are you deploying your application? (if relevant)

Custom

**Comment by JasonA-work** [comment]
Just incase this helps, here's how I solved this problem:

1. Figtree is my main font (English) and Noto Sans Thai is for when folks navigate to my Thai routes. Import the fonts as instructed in the docs. (In my case, I put this code in a separate Font.jsx file but I believe you can just put it in the layout.tsx if you'd like).

import { Figtree, Noto_Sans_Thai } from "next/font/google"

export const figTree = Figtree({
  subsets: ["latin"],
  display: "swap",
  adjustFontFallback: false,
  fallback: ["Roboto", "system-ui"],
  variable: "--figtree", //this is important and is what will enable this to work
})

export const notoSansThai = Noto_Sans_Thai({
  subsets: ["latin", "thai"],
  display: "swap",
  adjustFontFallback: false,
  fallback: ["Roboto", "system-ui"],
  variable: "--notoSansThai", //this is important and is what will enable this to work
  preload: false
})

2. In the root folder's layout.tsx file, in the <html> tag, assign the className as such:
<html lang="en" className={`${notoSansThai.variable} ${figTree.variable}`}>

3. In my css, here's the code that assigns the variable:

html:lang(en),
:lang(en) {
  font-family: var(--figtree);
}

html:lang(th),
:lang(th) {
  font-family: var(--notoSansThai);
}

Basically, if the html tag has the property "en" for the lang attribute, it'll use figtree. Similarly, if the html tag has the property "th", it'll use notoSansThai. Sometimes, I want specific elements to use the non-default language, so I have the rule for ":lang(en)" and ":lang(th)" too. This one, I manually assign the lang attribute to that particular DOM element.

I hope this helps. Let me know if you have any questions :)

**Comment by sebastiangrebe** [comment]
Ok that would work fine if I would want to specify certain elements to use one specific font but I don't want to specify this on purpose as this is technically not needed.

This is my earlier font-face config which handled this just simply by the browser capability of identifying if a font can handle all charachters of a string:

```
const fontFaceCss = fontPath => css`
  @font-face {
    font-family: proxima-nova;
    src: local('proxima-nova'),
      url('${fontPath}/Proxima-Nova-Bold.woff2') format('woff2'),
      url('${fontPath}/Proxima-Nova-Bold.woff') format('woff'),
      url('${fontPath}/Proxima-Nova-Bold.otf') format('opentype'),
      url('${fontPath}/Proxima-Nova-Bold.ttf') format('truetype'),
      url('${fontPath}/Proxima-Nova-Bold.eot?') format('eot');
    font-weight: 700;
    font-style: normal;
    font-display: swap;
  }

  @font-face {
    font-family: proxima-nova;
    src: local('proxima-nova'),
      url('${fontPath}/Proxima-Nova-Light.woff2') format('woff2'),
      url('${fontPath}/Proxima-Nova-Light.woff') format('woff'),
      url('${fontPath}/Proxima-Nova-Light.otf') format('opentype'),
      url('${fontPath}/Proxima-Nova-Light.ttf') format('truetype'),
      url('${fontPath}/Proxima-Nova-Light.eot?') format('eot');
    font-weight: 300;
    font-style: normal;
    font-display: swap;
  }

  @font-face {
    font-family: proxima-nova;
    src: local('proxima-nova'),
      url('${fontPath}/Proxima-Nova-Regular-Italic.woff2') format('woff2'),
      url('${fontPath}/Proxima-Nova-Regular-Italic.woff') format('woff'),
      url('${fontPath}/Proxima-Nova-Regular-Italic.otf') format('opentype'),
      url('${fontPath}/Proxima-Nova-Regular-Italic.ttf') format('truetype'),
      url('${fontPath}/Proxima-Nova-Regular-Italic.eot?') format('eot');
    font-weight: 400;
    font-style: italic;
    font-display: swap;
  }

  @font-face {
    font-family: proxima-nova;
    src: local('proxima-nova'),
      url('${fontPath}/Proxima-Nova-Regular.woff2') format('woff2'),
      url('${fontPath}/Proxima-Nova-Regular.woff') format('woff'),
      url('${fontPath}/Proxima-Nova-Regular.otf') format('opentype'),
      url('${fontPath}/Proxima-Nova-Regular.ttf') format('truetype'),
      url('${fontPath}/Proxima-Nova-Regular.eot?') format('eot');
    font-weight: 400;
    font-style: normal;
    font-display: swap;
  }

  @font-face {
    font-family: proxima-nova;
    src: local('proxima-nova'),
      url('${fontPath}/Proxima-Nova-Semibold.woff2') format('woff2'),
      url('${fontPath}/Proxima-Nova-Semibold.woff') format('woff'),
      url('${fontPath}/Proxima-Nova-Semibold.otf') format('opentype'),
      url('${fontPath}/Proxima-Nova-Semibold.ttf') format('truetype'),
      url('${fontPath}/Proxima-Nova-Semibold.eot?') format('eot');
    font-weight: 600;
    font-style: normal;
    font-display: swap;
  }

  @font-face {
    font-family: proxima-nova;
    src: local('proxima-nova'),
      url('${fontPath}/Proxima-Nova-Thin-Italic.woff2') format('woff2'),
      url('${fontPath}/Proxima-Nova-Thin-Italic.woff') format('woff'),
      url('${fontPath}/Proxima-Nova-Thin-Italic.otf') format('opentype'),
      url('${fontPath}/Proxima-Nova-Thin-Italic.ttf') format('truetype'),
      url('${fontPath}/Proxima-Nova-Thin-Italic.eot?') format('eot');
    font-weight: 200;
    font-style: italic;
    font-display: swap;
  }

  @font-face {
    font-family: proxima-nova;
    src: local('proxima-nova'),
      url('${fontPath}/Proxima-Nova-Thin.woff2') format('woff2'),
      url('${fontPath}/Proxima-Nova-Thin.woff') format('woff'),
      url('${fontPath}/Proxima-Nova-Thin.otf') format('opentype'),
      url('${fontPath}/Proxima-Nova-Thin.ttf') format('truetype'),
      url('${fontPath}/Proxima-Nova-Thin.eot?') format('eot');
    font-weight: 200;
    font-style: normal;
    font-display: swap;
  }

  @font-face {
    font-family: GESS;
    font-weight: 200;
    font-style: normal;
    font-variant: normal;
    src: local('GESS'),
      url('${fontPath}/GE%20SS%20Two%20Light.WOFF') format('woff');
    font-display: swap;
  }

  @font-face {
    font-family: GESS;
    font-weight: 300;
    font-style: normal;
    font-variant: normal;
    src: local('GESS'),
      url('${fontPath}/GE%20SS%20Two%20Light.WOFF') format('woff');
    font-display: swap;
  }

  @font-face {
    font-family: GESS;
    font-weight: 400;
    font-style: normal;
    font-variant: normal;
    src: local('GESS'),
      url('${fontPath}/GE%20SS%20Two%20Light.WOFF') format('woff');
    font-display: swap;
  }

  @font-face {
    font-family: GESS;
    font-weight: 600;
    font-style: normal;
    font-variant: normal;
    src: local('GESS'),
      url('${fontPath}/GE%20SS%20Two%20Medium.WOFF') format('woff');
    font-display: swap;
  }

  @font-face {
    font-family: GESS;
    font-weight: 700;
    font-style: normal;
    font-variant: normal;
    src: local('GESS'),
      url('${fontPath}/GE%20SS%20Two%20Bold.WOFF') format('woff');
    font-display: swap;
  }
`;
```

Also in your config you are `adjustFontFallback: false` which slightly defeats the purpose as your CLS will not benefit as much from next/font as it could.

**Comment by JasonA-work** [comment]
I have a perfect score and pretty much 0 CLS for my websites using this method even with adjustFontFallback being false, so I don't need to optimize it any further. Goodluck with getting to a solution your happy with :)

**Comment by taromorimoto** [comment]
I also have a similar problem where the fallback font is always Arial. I have two own local fonts. @sebastiangrebe did you ever figure out how to fix the problem?
